cockroachdb/datadog_checks/cockroachdb/metrics.py

```diff
@@ -0,0 +1,268 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+TRACKED_METRICS = {
```

Comment:
what's the difference between these two sets of metrics?

Reply:
We have to use/cache these for post-processing